### PR TITLE
feat: render .md files from packages

### PR DESCRIPTION
**Pull request by alexiscolin**
This PR aims to solves #2494 in order to parse `markdown` files from packages templates. It currently uses the file extension (from `window.location`) to detect if the content is a `md` file in a package template. 

_Although keeping the extension in the url is not a good practice for SEO & UX (because we are using pretty URLs for some other contents and are not consistent throughout gnoweb), This should do the work for now as short term solution since we are gonna parse `.md` from `go` soon with the new gnoweb design/codebase refactor (cc @gfanton)._

It also improves the package file template by removing global JS and pushing the `source` content into the main rendering flow from `renderer.js`.

### CODE REVIEW

**Comment by moul** [comment]
Hey Alexis, thank you!

Could you please share some before and after screenshots when making UX changes? We're hoping to have automatic deployment with @gfanton's upcoming refactor and @sw360cab's scripts. Until then, we'll need to manually share the changes to avoid requiring everyone to install and run the tool.

**Comment by alexiscolin** [comment]
Hey @moul 👋 
Sure! 

## Here is the current version (live version from [gno.land](https://gno.land/r/demo/boards/README.md)) of a `ReadMe` file from `package` template.

gno land_r_demo_boards_README md

## And here is the new version from the current PR (not in `code` block but markdown parsed by `MarkedJs`)

127 0 0 1_8888_r_demo_boards_README md

## Other pages keep the same design (code and markdown). For example:

127 0 0 1_8888_p_demo_grc_grc721_grc721_metadata gno

**Review by moul — APPROVED** [review]
LGTM for the result, but I don't have the skill to review the JS.

File: gno.land/pkg/gnoweb/static/js/renderer.js [hunk 1]
```diff
@@ -7,27 +7,40 @@ function renderUsernames(raw) {
...
+    const { markedHighlight } = globalThis.markedHighlight;
+    const { Marked } = globalThis.marked;
+    const markedInstance = new Marked(
+      markedHighlight({
+        langPrefix: "language-",
+        highlight(code, lang, info) {
+          if (lang === "json") {
+            try {
+              code = JSON.stringify(JSON.parse(code), null, 2);
+            } catch {}
```
Comment (zivkovicmilos) [inline gno.land/pkg/gnoweb/static/js/renderer.js:31]: Why don't we at least log the error?

Comment (zivkovicmilos) [reply gno.land/pkg/gnoweb/static/js/renderer.js]: Resolved in https://github.com/gnolang/gno/pull/2540/commits/c7800b80f8c219c44351c076d6beb112576daeed

**Pull request merged**